src/Extensions/BlogSitemapPattern.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Blog\Extensions;
+
+use Blog\Services\BlogService;
+use Plenty\Modules\Plugin\Events\LoadSitemapPattern;
+use Plenty\Modules\Plugin\Services\PluginSeoSitemapService;
+use Blog\AssistantServices\AssistantsService;
+
+
+class BlogSitemapPattern
+{
+    /**
+     * @param LoadSitemapPattern $sitemapPattern
+     */
+    public function handle(LoadSitemapPattern $sitemapPattern)
+    {
+        /** @var BlogService $blogService */
+        $blogService = pluginApp(BlogService::class);
+
+        /** @var PluginSeoSitemapService $seoSitemapService */
+        $seoSitemapService = pluginApp(PluginSeoSitemapService::class);
+
+        /** @var AssistantsService $assistantsService */
+        $assistantsService = pluginApp(AssistantsService::class);
+        $dynamoPosts = $assistantsService->getDynamoDbPosts();
+        $result = [];
+        foreach($dynamoPosts as $post) {
+            $url = $blogService->buildFullPostUrl($post);
```

Comment:
Show only already active and published posts

Reply:
check if the post ist for the correct client

Reply:
The active and published posts have been filtered, the available posts are only for the current client so the second comment is not applicable